build-system/tasks/e2e/package.json

```diff
@@ -6,7 +6,7 @@
   "devDependencies": {
     "@babel/register": "7.15.3",
     "babel-regenerator-runtime": "6.5.0",
-    "chromedriver": "95.0.0",
+    "chromedriver": "99.0.0",
```

Comment:
I don't think you want to commit this.

Reply:
undid it. (although i do think we need a separate PR to update this version because running e2e locally complains that the chromedriver version is mismatched with my local install of chrome?